patches/minecraft/net/minecraft/client/gui/screen/MainMenuScreen.java.patch

```diff
@@ -38,7 +38,7 @@
  
           this.field_230706_i_.func_110434_K().func_110577_a(field_194400_H);
           func_238463_a_(p_230430_1_, j + 88, 67, 0.0F, 0.0F, 98, 14, 128, 16);
-+         net.minecraftforge.client.ForgeHooksClient.renderMainMenu(this, p_230430_1_, this.field_230712_o_, this.field_230708_k_, this.field_230709_l_);
++         net.minecraftforge.client.ForgeHooksClient.renderMainMenu(this, p_230430_1_, this.field_230712_o_, this.field_230708_k_, this.field_230709_l_, l);
```

Comment:
Why is here 1 supplied but in the default overload later -1?

This does not make sense.

Reply:
It's `l` not `1`

Reply:
aaaaah
it is an l okey.
Perfect thank you.